### PR TITLE
Category and Predicate entailment

### PR DESCRIPTION
**Pull request by cbizon**
Committing the outcomes of Architecture call discussion.

1. The SRI will build a tool to help and 
2. It will be used in the smartapi registry's query

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,10 @@ This README documents the current strawman architecture.  Changes must be made v
     1. The SmartAPI Registry will provide a unified query system, returning information about all three API methods.  This query system will allow ARAs to locate the appropriate KPs.
     1. SRI will guarantee that metadata standards across the components allow such a unified query system.
     1. The SmartAPI registry will allow components to find all KPs by querying for biolink predicates. The SmartAPI registry will allow components to query by predicate from biolink-understood vocabularies, and return KPs that provide such metadata.
+1. Query graphs entering the system may be meaningfully the same as information stored in KPs, but the representation may be different, for instance using an inverse predicate, or using an subclass of an entity type, or a subpredicate.
+     1. This is primarily the ARA's responsibility to handle, but to aid ARAs
```
Comment (patrickkwang) [inline README.md:66]:
```suggestion
     1. This is primarily the ARA's responsibility to handle, but to aid ARAs:
```

File: README.md [hunk 2]
```diff
@@ -62,6 +62,10 @@ This README documents the current strawman architecture.  Changes must be made v
...
+          1. The SRI will implement a library that produces class-level triples that are logically entailed by an input class-level triple
```
Comment (patrickkwang) [inline README.md:67]: My interpretation of "logically entailed" is that this library will move _up_ the node-category and edge-predicate hierarchies, but not down. For example `biolink:Disease-[biolink:affects]->biolink:ChemicalSubstance` → `biolink:DiseaseOrPhenotypicFeature-[biolink:related_to]->biolink:ChemicalSubstance`, but not the inverse?

**Review by marcdubybroad — APPROVED** [review]

**Review by marcdubybroad — APPROVED** [review]
Does anyone have a handy link to a visual tool to view the predicate tree? Similar to what created the visual representation on this page:
https://docs.google.com/presentation/d/1EYYX9ZjirtPsDQXlFTmbFfwQjkvQFMgbcMK4UH2kKzQ/edit#slide=id.ga32599de0e_0_151

**Comment by balhoff** [comment]
That is a screenshot of Protégé desktop: https://protege.stanford.edu

You could open the OWL version of biolink-model in Protégé and and view the Object Property hierarchy.

https://github.com/biolink/biolink-model/blob/master/biolink-model.owl.ttl

**Comment by cbizon** [comment]
Closing this PR and replacing it.

**Pull request closed without merging**